lib/client/bst-config.ts

```diff
@@ -112,7 +112,7 @@ export class BSTConfig {
         const pipeInfo = await BSTConfig.createSpokesPipe();
 
         return {
-            "sourceID": pipeInfo.endpoint.name,
+            "sourceID": pipeInfo.endPoint.name,
```

Comment:
Just curious, is this a tslint recommendation?, otherwise we might want to only camelcase not compound words 🙂 - ref: https://softwareengineering.stackexchange.com/a/230165